src/DynamoCore/Engine/EngineController.cs

```diff
@@ -586,11 +613,20 @@ private void LibraryLoaded(object sender, LibraryServices.LibraryLoadedEventArgs
 
         #region Implement IAstNodeContainer interface
 
+        /// <summary>
+        /// Indicates to start compiling a NodeModel to AST nodes.
```

Comment:
This class represents the intermediate state (Compiling state) when a nodemodel is compiled to AST nodes